test/integration/targets/docker_swarm_service/tasks/tests/options.yml

```diff
@@ -959,10 +959,10 @@
     name: "{{ service_name }}"
     image: alpine:3.8
     publish:
-      - protocol: tcp
+      - protocol: "tcp"
         published_port: 60001
         target_port: 60001
-      - protocol: udp
+      - protocol: "udp"
```

Comment:
There's no need to quote this; YAML doesn't convert `udp` into something non-stringy :)